internal/etl/transform/transform.go

```diff
@@ -76,6 +82,22 @@ func (t Transformer) Transform(records []RawRecord, collection config.GeoSpatial
 	return result, nil
 }
 
+func (t Transformer) generateFieldValuesSubstitutions(fieldValuesByName map[string]any, substitutionFile string) ([]map[string]any, error) {
```

Comment:
Denk dat deze func ook in `extend-values.go` kan, aangezien het geen afhankelijkheden op data uit deze file heeft.

Reply:
ok, ga ik doen. Probeerde een beetje de code style te volgen, maar had verder geen inhoudelijke reden.